documentation/staging/content/userguide/managing-domains/domain-lifecycle/startup.md

```diff
@@ -227,6 +230,8 @@ the operator will not override the environment variable based on the shutdown co
 | `SHUTDOWN_TYPE` | `Graceful` | `Graceful` or `Forced` |
 | `SHUTDOWN_TIMEOUT` | 30 | Whole number in seconds where 0 means no timeout |
 | `SHUTDOWN_IGNORE_SESSIONS` | `false` | Boolean indicating if active sessions should be ignored; only applicable if shutdown is graceful |
+| `SHUTDOWN_WAIT_FOR_ALL_SESSIONS` | `false` | `true` to wait for all HTTP sessions during inflight work handling; `false` to wait for non-persisted HTTP sessions only ; only applicable if shutdown is graceful |
+
```

Comment:
What happens when a customer configures both serverPod.waitForAllSessions and SHUTDOWN_WAIT_FOR_ALL_SESSIONS?

Reply:
In other words, the documentation states `When a user-configured environment variable is present,
the operator will not override the environment variable based on the shutdown configuration.` ; so does this hold true for the new setting?

Reply:
For the new property, I just followed the current pattern as the other shutdown properties (type, timeout and ignoreSessions) that already existed.  We only add the environment variable if it doesn't already exist.  In PodStepContext.updateEnvForShutdown():

Shutdown shutdown = getShutdownSpec();
addDefaultEnvVarIfMissing(env, "SHUTDOWN_TYPE", shutdown.getShutdownType());
addDefaultEnvVarIfMissing(env, "SHUTDOWN_TIMEOUT", String.valueOf(shutdown.getTimeoutSeconds()));
addDefaultEnvVarIfMissing(env, "SHUTDOWN_IGNORE_SESSIONS", String.valueOf(shutdown.getIgnoreSessions()));
addDefaultEnvVarIfMissing(env, "SHUTDOWN_WAIT_FOR_ALL_SESSIONS",String.valueOf(shutdown.getWaitForAllSessions()));

Reply:
Then this (and the other env vars), should mention that they override the corresponding domain resource field when explicitly set...

Reply:
In short, this confirms that the precedence is following the already documented convention for all env vars.  Thanks.